apps/ui/lib/lens/full/SupportThread/index.tsx

```diff
@@ -21,13 +21,8 @@ const mapStateToProps = (state) => {
 
 const mapDispatchToProps = (dispatch) => {
 	return {
-		actions: bindActionCreators(
-			pick(actionCreators, [
-				'addChannel',
-				'getActor',
-				'loadMoreViewData',
-				'removeChannel',
-			]),
+		actions: redux.bindActionCreators(
+			_.pick(actionCreators, ['addChannel', 'getActor', 'removeChannel']),
```

Comment:
Not in the context of this pr, but why do we pick certain props from `actionCreators`, we could just:

```javascript
actions: redux.bindActionCreators(
  actionCreators,
  dispatch,
)
```

Reply:
I have no good reason for the `pick`, we should probably stop doing it just to make things simpler